<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>房管封禁助手</string>
  </property>
  <widget class="QLineEdit" name="CookieInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitCookieBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>提交Cookie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddBlackBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加封禁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveBlackBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>移除封禁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BlackUIDInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名/UID：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="BlackTimeInput">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>封禁时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>520</width>
     <height>310</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>UID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>封禁时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>解封时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作员</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>使用登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RoomInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RoomBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>连接房间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Login">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>登录状态：未登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
